rust/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "bumble"
```

Comment:
we should consider squatting this package name on `crates.io`

Reply:
+1

Reply:
I'll improve the readme and add cargo metadata and any missing docs to make this ready for at least a name-squatting release.